packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -234,7 +235,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         :
         <Callout
           isBeakVisible={ false }
-          className={ css('ms-Dropdown-callout', styles.callout) }
+          className={ css('ms-Dropdown-callout', styles.callout,
+            calloutProperties ? calloutProperties.className : undefined) }
```

Comment:
@mikewheaton, Here is an example https://github.com/OfficeDev/office-ui-fabric-react/blob/4d2ef75edb41b60f6d88bbf901e229ce2b04b3cd/packages/office-ui-fabric-react/src/components/Button/Button.Props.ts
It's passing IconProps object, ContextualMenuProps to Button